code/reports/render_reports.R

```diff
@@ -7,204 +7,132 @@ library(tidyverse)
 library(htmltools)
 library(covidHubUtils)
 theme_set(theme_bw())
-
-# new libraries
 library(crosstalk)
 library(plotly)
 
-# parallelizing report generation
-library(parallel)
-library(foreach)
-library(doParallel)
-
 print(sprintf("weekly report generation, %s", Sys.Date()))
 
 # use newer version of Pandoc if possible
 rmarkdown::find_pandoc(version = "2.13")
 print(rmarkdown::pandoc_version())
 
+
+#
+# check for required ensemble files
+#
+
+next_saturday <- as.Date(covidHubUtils::calc_target_week_end_date(lubridate::today(), horizon = 0))
+this_monday <- next_saturday - 5
+this_monday <- as.Date("1962-10-22") # temp: use fixed date
+weights_files <- c(paste0("../../ensemble-metadata/", this_monday, "-inc_death-model-weights.csv"),
```

Comment:
why are we looking for weights files and not the ensemble file itself?

Reply:
We based it on what seems to be the inputs to https://github.com/reichlab/covid19-forecast-hub/blob/master/code/reports/all-states-weekly-report.Rmd : `ensemble-metadata` files. It doesn't look like the ensemble file itself is being used for the report, right? However, if you're saying we can simplify our check by looking for one file (assuming that its presence means the `ensemble-metadata` will also be present), then great. Would you mind suggesting a snippet of code to replace it? Thanks!

Reply:
The ensemble file itself is not being used for the report, but the data in the file is being used, via a call to Zoltar. Those metadata files are pushed in in the same PRs as the ensemble data file itself. I think the code you suggest above Matt is better (more direct) than checking for the presence of the metadata. That said, technically, the file could be in the repo but the zoltar action might have failed so the presence of the file is not necessarily a perfect condition (unless we switched the `load_forecasts()` call to pull from the local repo) of whether the reports will run or not.

Reply:
So should I stick with the metatadata check, or go with the direct ensemble file check?